src/dataregistry/schema/schema.yaml

```diff
@@ -0,0 +1,282 @@
+    nullable: False
+  db_version_major:
+    type: "Integer"
+    description: "Major version of schema"
+    nullable: False
+  db_version_minor:
+    type: "Integer"
+    description: "Minor version of schema"
+    nullable: False
+  db_version_patch:
+    type: "Integer"
+    description: "Patch version of schema"
+    nullable: False
+  git_hash:
+    type: "String"
+    description: "Git hash at time of schema creation"
+  repo_is_clean:
+    type: "Boolean"
+    description: "Was git repo clean at schema creation?"
+  update_method:
+    type: "String"
+    description: "What type of schema update does this entry relate to ('CREATE','MODIFY','MIGRATE')"
+    nullable: False
+  schema_enabled_date:
+    type: "DateTime"
+    description: "Date schema was created/updated"
+    nullable: False
+  comment:
+    type: "StringLong"
+    description: "Comment relating to new provenance entry"
+
+execution_alias:
+  execution_alias_id:
+    description: "Unique identifier for execution alias"
+    type: "Integer"
+    primary_key: True
+  supersede_date:
+    type: "DateTime"
+    description: "If a new entry has been added to the table with the same alias name (but different `dataset_id`), the old entry will be superseded. `supersede_date` in the old entry tracks when this happened. If the entry has not been superseded, `supersede_date` will be None"
+  creator_uid:
+    type: "StringShort"
+    description: "UID of person who registered the entry"
+    nullable: False
+  register_date:
+    type: "DateTime"
+    description: "Date the execution alias was registered"
+    nullable: False
+  alias:
+    type: "String"
+    description: "User given execution alias name"
+    nullable: False
+  execution_id:
+    type: "Integer"
+    foreign_key: True
+    foreign_key_schema: "self"
+    foreign_key_table: "execution"
+    foreign_key_row: "execution_id"
+    description: "Execution this alias is linked to"
+
+dataset_alias:
+  dataset_alias_id:
+    description: "Unique identifier for dataset alias"
+    type: "Integer"
+    primary_key: True
+  supersede_date:
+    type: "DateTime"
+    description: "If a new entry has been added to the table with the same alias name (but different `dataset_id`), the old entry will be superseded. `supersede_date` in the old entry tracks when this happened. If the entry has not been superseded, `supersede_date` will be None"
+  creator_uid:
+    type: "StringShort"
+    description: "UID of person who registered the entry"
+    nullable: False
+  register_date:
+    type: "DateTime"
+    description: "Date the dataset alias was registered"
+    nullable: False
+  alias:
+    type: "String"
+    description: "User given dataset alias name"
+    nullable: False
+  dataset_id:
+    type: "Integer"
+    foreign_key: True
+    foreign_key_schema: "self"
+    foreign_key_table: "dataset"
+    foreign_key_row: "dataset_id"
+    description: "Dataset this alias is linked to"
+
```

Comment:
Done

Reply:
In case something like this comes up again: I was actually thinking of a yaml comment preceding the table definition, e.g.
```# For each row in this table, exactly one of input_id and input_production_id is non-null```
but the way you've done it is perfectly ok, perhaps preferable.